src/components/layout.tsx

```diff
@@ -1,22 +1,19 @@
 import * as React from "react"
-import Header from "./header"
-import Footer from "./footer"
-import Head from "./head"
 import "../styles.css"
-
+import { Slice } from "gatsby"
 interface LayoutProps {
```

Comment:
You'll need to change the props since they were intended for the Head component. They are no longer necessary